### PR TITLE
Print runtime error message

### LINKED ISSUE #23: Runtime error (15) is not reported properlty
For python we can easily get this error, it'll be nice to see the error message.

### PR DESCRIPTION
**Pull request by jhshi**
Fix #23

### CODE REVIEW

File: leetcode/views/result.py [hunk 1]
```diff
@@ -102,6 +106,51 @@ def make_compile_error_view(self):
...
+    def make_timeout_view(self):
+        blank = urwid.Divider()
+        status_header = urwid.AttrWrap(urwid.Text('Run Code Status: '), 'body')
+        status = urwid.AttrWrap(urwid.Text('Time Limit Exceeded'), 'hometag')
+        columns = urwid.Columns([(17, status_header), (20, status)])
+        column_wrap = urwid.WidgetWrap(columns)
+        result_header = urwid.Text('--- Run Code Result: ---', align='center')
+        your_input_header = urwid.Text('Last executed input:')
+        your_input = urwid.Text(self.result['last_testcase'])
+        your_answer_header = urwid.Text('')
```
Comment (chishui) [inline leetcode/views/result.py]: "answer" and "expected_answer" related UI element can be removed

File: leetcode/views/result.py [hunk 2]
```diff
@@ -102,6 +106,51 @@ def make_compile_error_view(self):
...
+    def make_runtime_error_view(self):
+        blank = urwid.Divider()
+        status_header = urwid.AttrWrap(urwid.Text('Run Code Status: '), 'body')
+        status = urwid.AttrWrap(urwid.Text('Runtime Error'), 'hometag')
+        columns = urwid.Columns([(17, status_header), (20, status)])
+        column_wrap = urwid.WidgetWrap(columns)
+        result_header = urwid.Text('--- Run Code Result: ---', align='center')
+        your_input_header = urwid.Text('Last input:')
+        your_input = urwid.Text(self.result['last_testcase'])
+        your_answer_header = urwid.Text('Error:')
```
Comment (chishui) [inline leetcode/views/result.py]: We'd better use the same title as leetcode display. And runtime error should appear before test cases?

Comment (jhshi) [reply leetcode/views/result.py]: Updated UI
Leetcode:
image
Terminal:
image

File: leetcode/views/result.py (hunk 1 above)
Comment (jhshi) [reply leetcode/views/result.py]: Fixed. 

Leetcode:
image

Terminal:
image

**Pull request merged**